### DISCUSSION TITLE
Lesson 1 Error

### DISCUSSION THREAD

**Discussion by Friend-Forest** [smartcontractkit/full-blockchain-solidity-course-py#1391] [Q&A] [answered]
`pragma solidity ^0.6.0;

```Solidity
contract SimpleStorage {

    // this will get intialized to 0
    uint256 public favoriteNumber;
   
    function store(unit256 _favoriteNumber) public {
       favoriteNumber = _favoriteNumber;
    }
}`

```
I'm recieving this error when I try to compile the code.

contracts/Simple Storage.sol:8:20: DeclarationError: Identifier not found or not unique.
function store(unit256 _favoriteNumber) public {
^-----^

**Comment by cromewar** [comment]
Tottaly missed this @Friend-Forest 
Have you managed to solve this?

**Comment by pygpt** [comment, marked as answer]
you typed "unit256" instead of "uint256"
